pandas/tests/io/test_pytables.py

```diff
@@ -1223,6 +1223,16 @@ def test_append_all_nans(self):
             reloaded = read_hdf(path, 'df_with_missing')
             tm.assert_frame_equal(df_with_missing, reloaded)
 
+    def test_read_missing_key_close_store(self):
+        # GH 25766
+        with ensure_clean_path(self.path) as path:
+            df = pd.DataFrame({'a': range(2), 'b': range(2)})
+            df.to_hdf(path, 'k1')
+
+            pytest.raises(KeyError, pd.read_hdf, path, 'k2')
```

Comment:
this is not an accepted format any longer, use the context manager

```
with pytest.raises(....):
    ...
```

Reply:
updated https://github.com/pandas-dev/pandas/pull/25863/commits/244cab4ecb1d27564580a5702a93f58235075173 . But with form `pytest.raises(KeyError, ...)` I followed the codestyle in this test file. What about some unification of this style across test files in future?

Reply:
yep this happens to be the last file - i think it’s fixed in another PR